x/group/spec/02_state.md

```diff
@@ -0,0 +1,80 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `group` module uses the `orm` package which provides table storage with support for
+primary keys and secondary indexes. `orm` also defines `Sequence` which is a persistent unique key generator based on a counter that can be used along with `Table`s.
+
+Here's the list of tables and associated sequences and indexes stored as part of the `group` module.
+
+## Group Table
+
+The `groupTable` stores `GroupInfo`: `0x0 | []byte(group_id) -> ProtocolBuffer(GroupInfo)`
+
+### groupSeq
+
+The value of `groupSeq` is incremented when creating a new group and corresponds to the new `GroupId`: `0x1 | 0x1 -> BigEndian`
```

Comment:
Do we have docs on the ORM module? If yes, might be useful to link those, so that the second `0x1` doesn't come here arbitrarily.

Reply:
No there's no doc for the ORM module yet afaik, I have created an issue to track that: #320
In the mean time, I can add a comment to explain that this second `0x1` corresponds to the ORM `sequenceStorageKey`